controllers/actions.github.com/resourcebuilder.go

```diff
@@ -19,14 +20,92 @@ const (
 	jitTokenKey = "jitToken"
 )
 
-// labels applied to resources
+// Labels applied to resources
 const (
-	LabelKeyAutoScaleRunnerSetName      = "auto-scaling-runner-set-name"
-	LabelKeyAutoScaleRunnerSetNamespace = "auto-scaling-runner-set-namespace"
+	// Kubernetes labels
+	LabelKeyKubernetesPartOf    = "app.kubernetes.io/part-of"
+	LabelKeyKubernetesComponent = "app.kubernetes.io/component"
+	LabelKeyKubernetesVersion   = "app.kubernetes.io/version"
+
+	// Github labels
+	LabelKeyGithubScaleSetName      = "actions.github.com/scale-set-name"
```

Comment:
I would name is as `runner-scale-set-name` to match the real name.
you will need to modify the E2E test workflow since it was looking at the listener pod via the label.

Reply:
The reason why I changed this is to conform to our [ADR](https://github.com/actions/actions-runner-controller/blob/master/adrs/2023-04-14-adding-labels-k8s-resources.md#listener)
If we move back to `runner-scale-set-name`, ideally I'd like to update ADR to specify that :relaxed: